Front-end/parquick/flow-typed/npm/jsonwebtoken_v8.5.x.js

```diff
@@ -0,0 +1,167 @@
+// flow-typed signature: edd16f7dc143fae597115e4223dc24dd
```

Comment:
Let's try making flow-typed type definitions into their own pull request in the future so they don't pollute the rest of your changes in the PR.